### PR TITLE
Tag FLAC.jl v0.2.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [dmbates/FLAC.jl](https://github.com/dmbates/FLAC.jl)
Release: [v0.2.1](https://github.com/dmbates/FLAC.jl/releases/tag/v0.2.1)
Travis: [Travis Build Status](https://travis-ci.org/dmbates/FLAC.jl/builds/237318553)
Diff: [vs v0.2.0](https://github.com/dmbates/FLAC.jl/compare/f96bdfc4f8d438bfac9cc41da2533a5702c675a3...57556704413f9e0edc4334050b526abd6720dfa5)
`requires` vs v0.2.0: 
```diff
--- 0.2.0/requires
+++ 0.2.1/requires
@@ -1,4 +1,5 @@
 julia 0.5
+Compat
 BinDeps
 FileIO
 @osx Homebrew
```
cc: @staticfloat

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

**Comment by staticfloat** [comment]
JuliaCIBot is sad because it can't `sudo apt-get install`.  :(

File: FLAC/versions/0.2.1/requires [hunk 1]
```diff
@@ -0,0 +1,6 @@
+julia 0.5
+Compat
```
Comment (tkelman) [inline FLAC/versions/0.2.1/requires]: It's best to declare a minimum version dependency that implements the features you need.

Comment (tkelman) [inline FLAC/versions/0.2.1/requires]: 0.17 for abstract type. regular type and immutable don't need compat, unless there's some call overloading needed on 0.4

**Comment by staticfloat** [comment]
I made a `v0.2.2` release to do that.  Thanks.

**Pull request closed without merging**